src/layer/AbstractDEMLayer.ts

```diff
@@ -25,21 +25,36 @@ export interface ConstructorParameters {
 
 export class AbstractDEMLayer extends AbstractSentinelHubV3Layer {
   protected demInstance: DEMInstanceType;
+  protected egm: boolean = null;
+  protected clampNegative: boolean = null;
 
-  public constructor({ demInstance, ...rest }: ConstructorParameters) {
+  public constructor({ demInstance, egm, clampNegative, ...rest }: ConstructorParameters) {
     super(rest);
     this.demInstance = demInstance;
+    this.egm = egm;
+    this.clampNegative = clampNegative;
   }
 
   public async updateLayerFromServiceIfNeeded(reqConfig?: RequestConfiguration): Promise<void> {
     await ensureTimeout(async innerReqConfig => {
       if (!(this.instanceId && this.layerId)) {
         return;
       }
-
-      if (!this.demInstance) {
+      //update properties defined on parent layer
+      await super.updateLayerFromServiceIfNeeded(innerReqConfig);
+      //update DEM specific properties if they're not set
+      if (!this.demInstance || isBooleanNull(this.egm) || isBooleanNull(this.clampNegative)) {
         const layerParams = await this.fetchLayerParamsFromSHServiceV3(innerReqConfig);
-        this.demInstance = layerParams['demInstance'] ? layerParams['demInstance'] : DEMInstanceType.MAPZEN;
+        if (!this.demInstance) {
+          this.demInstance = layerParams['demInstance'] ? layerParams['demInstance'] : DEMInstanceType.MAPZEN;
```

Comment:
I don't think it's a good idea to use default value. This could give false impression that the layer is defined in a certain way on the service. IMO all of them should be just set to `null` (also `clampNegative` and `egm`).

Reply:
Defaults were removed and check was added to ensure `fetchLayerParamsFromSHServiceV3` is not called everytime because of null values.